admin/gendoc.pl

```diff
@@ -28,8 +28,9 @@
 # GLOBAL CONFIGS
 #################################################################################################
 
-$g_currenttime=gmtime;
-$g_currentuser = getlogin || getpwuid($<) || "Unknown";
+$g_currenttime = gmtime($ENV{SOURCE_DATE_EPOCH} || time);
+$g_currentuser = ($ENV{SOURCE_DATE_EPOCH} && "reproducible") ||
```

Comment:
I think it should have a separate config knob for "reproducible build"; making username depend on some unrelated environment variable looks less than obvious

Reply:
OK, how about using the already existing `$USER` environment variable?